packages/frontend/src/pages/campaigns/filters/utils/index.ts

```diff
@@ -0,0 +1,38 @@
+import { Template } from "@carrot-kpi/sdk";
+
+export const CARROT_DEFAULT_TEMPLATE_ID = 1;
```

Comment:
I'm not sure I fully get what these are for. It should be the template id for the default selected checkboxes right? If that's the case, why is this a thing? I'd imagine the default state of the checkboxes on page render is for all of them to be selected (as we would show ALL the campaigns using any of the templates in that case).

Reply:
So basically we should show all by default and only filter when checked.

That makes sense. 

Yup, in that case this isn't needed. 

Thanks for pointing this out.

Reply:
LESGOOOOOOOOOOOo

Reply:
fixed: df180fc782eaa6d8a035701179412777fc890cba